src/anomalib/engine/engine.py

```diff
@@ -334,6 +435,30 @@ def test(
         Returns:
             _EVALUATE_OUTPUT: A List of dictionaries containing the test results. 1 dict per dataloader.
 
+        Examples:
+            # fit and test a one-class model
+            >>> from anomalib.engine import Engine
+            >>> from anomalib.models import Padim
+            >>> from anomalib.data import MVTec
+            >>> engine = Engine()
+            >>> model = Padim()
+            >>> datamodule = MVTec()
+            >>> model.learning_type
+            <LearningType.ONE_CLASS: 'one_class'>
+            >>> engine.fit(model, datamodule=datamodule)
+            >>> engine.test(model, datamodule=datamodule)
```

Comment:
Just to make it look better on the sphinx build
```suggestion
            # fit and test a one-class model
            >>> from anomalib.data import MVTec            
            >>> from anomalib.models import Padim            
            >>> from anomalib.engine import Engine

            >>> datamodule = MVTec()
            >>> model = Padim()            
            >>> model.learning_type
            <LearningType.ONE_CLASS: 'one_class'>

            >>> engine = Engine()
            >>> engine.fit(model, datamodule=datamodule)
            >>> engine.test(model, datamodule=datamodule)
```